alws/utils/multilib.py

```diff
@@ -249,9 +249,15 @@ async def update_module_index(module_index: IndexWrapper, module_name: str,
                                   module_stream: str, packages: list):
         if not packages:
             return
-        if module_index.has_devel_module() and 'python' not in module_name:
-            module = module_index.get_module(
-                f'{module_name}-devel', module_stream)
+        # Basically now we always will have devel module,
+        # but in case if not the original behavior is saved
+        if module_index.has_devel_module():
```

Comment:
I think, it's useless check now, we can safely remove it

Reply:
Fixed